lib/create-file-compression-interface-context.yml

```diff
@@ -1,12 +1,14 @@
 rule:
   meta:
-    name: open cabinet file
-    namespace: host-interaction/file-system
+    name: create File Compression Interface context
```

Comment:
Thoughts on adding `on Windows` to make it more clear?

Reply:
Thanks! Renamed the rule in https://github.com/mandiant/capa-rules/pull/808/commits/ac09516cca968e54d6b5764405337cba3f290c21